common/mqtt_client/tests/packet_size_tests.rs

```diff
@@ -40,16 +40,14 @@ async fn packet_size_within_limit() -> Result<(), anyhow::Error> {
 // This checks the mqtt packet size that exceeds the limit
 async fn packet_size_exceeds_limit() -> Result<(), anyhow::Error> {
     // Start the broker
-    let mqtt_server_handle = tokio::spawn(async {
-        start_broker_local("../../configuration/rumqttd/rumqttd_5884.conf").await
-    });
+    let _mqtt_server_handle =
+        tokio::spawn(async { rumqttd_broker::start_broker_local(58585).await });
```

Comment:
As below, make the number a `const` with name

Reply:
fixed